src/plugins/intel_gpu/src/kernel_selector/kernel_selector_params.h

```diff
@@ -627,6 +627,7 @@ struct fused_operation_desc {
 struct base_params : public Params {
     virtual ~base_params() {}
 
+    bool is_dynamic = false;
```

Comment:
Yes this can be added, but I think this can be set at the creation time of the dynamic impl ?